README.md

```diff
@@ -2,21 +2,21 @@
 
 This package models Pinterest Ads data from [Fivetran's connector](https://fivetran.com/docs/applications/pinterest-ads). It uses data in the format described by [this ERD](https://docs.google.com/presentation/d/1YMsP4fBwb0sGoOgDWfIEVVOkfXfljOseulgx9wC87qk/edit).
 
-The main focus of the package is to transform the core ad object tables into analytics-ready models, including an 'ad adapter' model that can be easily unioned in to other ad platform packages to get a single-view 
+This package transforms the core ad object tables into analytics-ready models, including an 'ad adapter' model that can be easily unioned in to other ad platform packages to get a single view.
 
 ## Models
 
-This package contains transformation models, designed to work simultaneously with our [Pinterest Ads source package](https://github.com/fivetran/dbt_pinterest_source). A depenedency on the source package is declared in this package's `packages.yml` file, so it will automatically download when you run `dbt deps`. The primary outputs of this package are described below.
+This package contains transformation models, designed to work simultaneously with our [Pinterest Ads source package](https://github.com/fivetran/dbt_pinterest_source). A dependency on the source package is declared in this package's `packages.yml` file, so it will automatically download when you run `dbt deps`. The primary outputs of this package are described below.
 
 | **model**                 | **description**                                                                                                             |
 | ------------------------- | --------------------------------------------------------------------------------------------------------------------------- |
-| pinterest_ads__ad_adapter | Each record represents the daily ad performance of each pin promotion, including information about the used UTM parameters. |
+| pinterest_ads__ad_adapter | Each record represents the daily ad performance of each pin promotion, including information about its UTM parameters. |
```

Comment:
I assume that you meant "the UTM parameters used by the pin promotion." If this assumption is incorrect, please let me know because my edit won't apply.